functions.php

```diff
@@ -40,7 +40,7 @@ function extractDataAndDisplay(array $data):string
 function executeFormCheckAndDataInsert(array $get): string
 {
     $message = '';
-    if (empty($_GET)) {
+    if (empty($get)) {
```

Comment:
You're still using `$_GET` here. It's probably fine to use `$_GET` instead of the variable you're passing in, because you're not testing this function. If you were testing it, or parts of it, I would tell you to pass it in and use it everywhere.
But it's fine. Change this back to `$_GET` and remove the `$get` parameter from your function (don't forget to take it out of the function call in your form page).